aten/src/ATen/core/type.cpp

```diff
@@ -29,7 +29,7 @@ std::ostream& operator<<(std::ostream & out, const Type & t) {
       }
       out << ")";
     }
-    if (value->autogradZero() && *value->autogradZero()) {
+    if (value->undefined() && *value->undefined()) {
       out << "[AutogradZero]";
```

Comment:
We should be consistent about names: either its undefined or autogradZero, mixing both is worse than one or the other.